validator-api/validator_api/services/scoring_service.py

```diff
@@ -266,19 +271,58 @@ async def get_detailed_video_description_embedding_score(self, video_id, task_ov
             return detailed_video_description, None, None
         return detailed_video_description, embedding, await self.get_description_uniqueness_score(embedding)
 
+    async def get_boosted_multiplier(self, focusing_task: str, focusing_description: str) -> float:
+        """
+        Get boosted tasks from the database "boosted_tasks" table
+        ask Gemini if the task matches any of the boosted tasks
+        return the multiplier for the task if it is boosted, otherwise 1.0
+        """
+        with get_db_context() as db:
+            boosted_tasks_records = db.query(BoostedTask).all()
+            # Convert to simple tuples containing just the data we need
+            boosted_tasks = [(task.title, task.description, task.multiplier) for task in boosted_tasks_records]
+        # print(f"Boosted tasks: {boosted_tasks}")
+        system_prompt = focus_scoring_prompts.BOOST_SCORING_SYSTEM_PROMPT.format(
```

Comment:
let's put a quick if statement here to just `return 1.0` if there are no boosted tasks. i.e. no need to prompt gemini if there aren't even any tasks to test on.